docs/index.md

```diff
@@ -44,9 +44,12 @@ The navigation bar's title reflects the name of the web map and the navigation b
 
 | Icon | Description |
 | ---- | ----------- |
-| ![Hamburger Drawer View](/docs/images/hamburger.png) | Ellipsis button to reveal or hide the app context drawer view. |
-| ![Zoom To Location](/docs/images/zoom-to-location.png) | Zoom to user's location. |
-| ![Add Feature](/docs/images/add-feature.png) | Add a new spatial feature to map. |
+| ![Bookmarks icon](/src/DataCollection.UWP/Assets/Calcite/bookmark-32.png) | Bookmarks button reveals list of bookmarks in the map |
+| ![TOC icon](/src/DataCollection.UWP/Assets/Calcite/layers-32.png) | Contents button shows a list of layers, which can be toggled on and off |
```

Comment:
Does the app make it clear this button is the "contents" button? If it's a direct TOC, perhaps we should just call it that?